lighting/material/add.glsl

```diff
@@ -9,7 +9,7 @@ license: MIT License (MIT) Copyright (c) 2024 Shadi EL Hajj
 #ifndef MATERIAL_ADD
 #define MATERIAL_ADD
 
-void add(Material a, Material b, Material r) {
+void materialAdd(Material a, Material b, Material r) {
```

Comment:
I love function overloading. There is plenty on LYGIA. Because of WGSL lack of it, I think we should be more careful and try to be more unique on the naming of the functions. 
This fact will make the WGSL version very different, but we can reduce the warm

Reply:
Agreed!